src/schemas.ts

```diff
@@ -0,0 +1,15 @@
+const search = {
+  params: {
+    keyword: {
+      type: 'string'
+    },
+    range: {
+      type: 'string',
+      enum: ['title', 'tag', 'all', 'author']
```

Comment:
Do you think it's possible to factor this out / use the same enum you had in `plugin`?